{
	"schema_version": "1.6.0",
	"id": "BIT-python-min-2023-33595",
	"modified": "2025-01-16T09:11:58.202055Z",
	"published": "2025-01-16T07:21:27.893Z",
	"aliases": [
		"BIT-python-2023-33595",
		"CVE-2023-33595",
		"PSF-2023-3"
	],
	"details": "CPython v3.12.0 alpha 7 was discovered to contain a heap use-after-free via the function ascii_decode at /Objects/unicodeobject.c.",
	"affected": [
		{
			"package": {
				"ecosystem": "Bitnami",
				"name": "python-min",
				"purl": "pkg:bitnami/python-min"
			},
			"severity": [
				{
					"type": "CVSS_V3",
					"score": "CVSS:3.1/AV:L/AC:L/PR:N/UI:R/S:U/C:N/I:N/A:H"
				}
			],
			"ranges": [
				{
					"type": "SEMVER",
					"events": [
						{
							"introduced": "3.12.0-alpha0"
						},
						{
							"fixed": "3.12.0-alpha8"
						}
					]
				}
			],
			"database_specific": {
				"source": "https://github.com/bitnami/vulndb/tree/main/data/python-min/BIT-python-min-2023-33595.json"
			}
		}
	],
	"references": [
		{
			"type": "WEB",
			"url": "https://github.com/python/cpython/issues/103824"
		},
		{
			"type": "WEB",
			"url": "https://github.com/python/cpython/pull/103993/commits/c120bc2d354ca3d27d0c7a53bf65574ddaabaf3a"
		}
	],
	"database_specific": {
		"cpes": [
			"cpe:2.3:a:python:python:*:*:*:*:*:*:*:*"
		],
		"severity": "Medium"
	}
}
